code/modules/mob/living/carbon/human/human.dm

```diff
@@ -48,6 +37,7 @@
 	var/mob/M = src
 	faction |= "\ref[M]" //what
 
+	prepare_data_huds()

```

Comment:
it seems `prepare_huds()` actually calls `prepare_data_huds()`, and `prepare_huds()` is covered in mob/New()